src/common/xo/index.js

```diff
@@ -1171,6 +1171,12 @@ export const createVgpu = (vm, { gpuGroup, vgpuType }) =>
 
 export const deleteVgpu = vgpu => _call('vm.deleteVgpu', resolveIds({ vgpu }))
 
+export const shareVm = vm =>
+  confirm({
+    title: _('shareVmModalTitle'),
+    body: _('shareVmModalMessage', { self: <strong>{vm.resourceSet}</strong> }),
```

Comment:
I think the user would want to see the resource set's name instead of its ID, even if it's not unique.